infra/scripts/docker_build_cross_arm_compiler_debug.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && echo "Please don't source ${BASH_SOURCE[0]}, execute it" && return
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_PATH="$CURRENT_PATH/../../"
+
+# prepare rootfs
+#if [ -z "$ROOTFS_DIR" ] || [ ! -d $ROOTFS_DIR ]; then
+#  echo "It will use default rootfs path"
+#else
+#  DOCKER_VOLUMES+=" -v $ROOTFS_DIR:/opt/rootfs"
+#  DOCKER_ENV_VARS+=" -e ROOTFS_DIR=/opt/rootfs"
+#fi
```

Comment:
```suggestion
# prepare rootfs
if [ -z "$ROOTFS_DIR" ] || [ ! -d $ROOTFS_DIR ]; then
  echo "It will use default rootfs path"
else
  DOCKER_VOLUMES+=" -v $ROOTFS_DIR:/opt/rootfs"
  DOCKER_ENV_VARS+=" -e ROOTFS_ARM=/opt/rootfs"
fi
```

For rootfs setting?
https://github.com/Samsung/ONE/blob/c06fe64f4c280e06a4ec7f5b0cc450d45393e067/infra/nncc/Makefile.arm32#L11

Reply:
Current build with `Makefile.arm32` doesn use arm root fs :)